clients/src/test/java/org/apache/kafka/common/requests/LeaveGroupResponseTest.java

```diff
@@ -165,4 +169,55 @@ public void testEqualityWithMemberResponses() {
             assertEquals(primaryResponse.hashCode(), reversedResponse.hashCode());
         }
     }
+
+    @ParameterizedTest
+    @ApiKeyVersionsSource(apiKey = ApiKeys.LEAVE_GROUP)
+    public void testNoErrorNoMembersResponses(short version) {
+        LeaveGroupResponseData data = new LeaveGroupResponseData()
+            .setErrorCode(Errors.NONE.code())
+            .setMembers(Collections.emptyList());
+
+        if (version < 3) {
+            assertThrows(UnsupportedVersionException.class,
+                    () -> new LeaveGroupResponse(data, version));
```

Comment:
nit: Could we indent this line with 4 spaces in order to remain consistent with the existing code?